core/color.py

```diff
@@ -0,0 +1,12 @@
+"""
+Contains definitions for game players.
+"""
+
+from enum import Enum
+
+class Color(Enum):
+    """
+    Indicates a game color, for e.g. turns and piece colors.
+    """
+    BLACK = 1
+    WHITE = 2
```

Comment:
can probably merge this into a game module along with some kind of abstract `Game` class modeled after our state representation `{board, turn}`